ISSUE_TEMPLATE.md

```diff
@@ -1,19 +1,24 @@
-- Parent issues: ...
-- Relevant issues: ...
+[//]: # "Check if the title is descriptive!"
+- Parent issues: -
+- Relevant issues: -
+- Estimated effort: PLACEHOLDER
 
 * * *
 
 ## User story
 The user story for the issue.
 
 ## What
-[Optional] Specify in more detail what this issue adds to the repository.
+Specify in more detail what this issue adds to the repository.
 
 ## Why
-[Optional] Specify in more detail why this issue is needed in the project.
+Specify in more detail why this issue is needed in the project.
 
 ## How
-[Optional] Specify, in general terms, how this feature can be added.
+Specify in general terms how this feature can be added.
+
+## Definition of done
+Specify what must be done exactly before this task is done.
```

Comment:
Not the clearest sentence imo, maybe: "Specify exactly what must be done before this task can be considered done"